apps/portals/src/Navbar.tsx

```diff
@@ -145,16 +146,19 @@ function Navbar() {
   // if the home route does not contain any titles, then just show a link
   const homeConfigTitleCount = homeRouteConfig?.synapseConfigArray?.filter(
     (config) => config.title !== undefined,
+    (config) => config.title !== undefined,
   ).length
   const isHomeDropdown = homeConfigTitleCount ? homeConfigTitleCount > 0 : false
   return (
     <React.Fragment>
-      <nav
+      <Box
+        component={'nav'}
         className={
           !showMenu
             ? 'flex-display nav top-nav'
             : 'flex-display nav top-nav mb-active'
         }
+        sx={RESPONSIVE_SIDE_PADDING}
```

Comment:
Shared style with the explore components, so the padding will necessarily be the same